go-controller/pkg/ovn/pods_test.go

```diff
@@ -1138,6 +1138,13 @@ var _ = ginkgo.Describe("OVN Pod Operations", func() {
 				gomega.Eventually(fakeOvn.controller.nbClient).Should(
 					libovsdbtest.HaveData(getExpectedDataPodsAndSwitches([]testPod{}, []string{"node1"})...))
 
+				// Once again, get pod from api with its metadata filled in
+				pod, err = fakeOvn.fakeClient.KubeClient.CoreV1().Pods(podTest.namespace).Get(context.TODO(), podTest.podName, metav1.GetOptions{})
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+				// Deleting port that no longer exists should be okay!
```

Comment:
I feel this comment is redundant, since the line below features that string.

But anyway, feel free to keep it.